.github/workflows/dm_chaos.yaml

```diff
@@ -120,10 +98,11 @@ jobs:
           cp -r $GITHUB_WORKSPACE/dm/chaos/cases/conf/ $GITHUB_WORKSPACE/bin/
           docker build -f $GITHUB_WORKSPACE/dm/chaos/manifests/Dockerfile -t dm:chaos $GITHUB_WORKSPACE/bin
           docker image list
+
       # Load DM docker image into KIND, see https://kind.sigs.k8s.io/docs/user/quick-start/#loading-an-image-into-your-cluster
-      # - name: Load DM docker image into KIND
-      #   run: |
-      #     kind load docker-image dm:chaos --name dm-chaos
+      - name: Load DM docker image into KIND
+        run: |
+          kind load docker-image dm:chaos
```

Comment:
default name for kind cluster is `chart-testing`, or we can configure with `cluster_name` when setup kind